src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -222,7 +223,8 @@
                            ssl-config)
                         :client-auth :none
                         :cipher-suites (or (:cipher-suites ssl-config) config/acceptable-ciphers)
-                        :protocols     (or (:protocols ssl-config) config/default-protocols)}))
+                        :protocols     (or (:protocols ssl-config) config/default-protocols)
+                        :cache-session-id (or (:cache-session-id ssl-config) true)}))
```

Comment:
I don't think `or` will work here. If `(:cache-session-id ssl-config)` is nil or false the `or` will evaluate to true.